### PR TITLE
EVG-6001: Change MongoDB Driver to go.mongodb.org/mongo-driver

### PR DESCRIPTION
**Pull request by tychoish**
(this is WIP)

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -87,16 +86,35 @@ type Settings struct {
...
-	err := db.FindOneQ(ConfigCollection, db.Query(byId(c.SectionId())), c)
-	if err != nil && err.Error() == errNotFound {
-		*c = Settings{}
-		return nil
+	env := GetEnvironment()
+	ctx, cancel := env.Context()
+	defer cancel()
+	coll := env.DB().Collection(ConfigCollection)
+
+	res := coll.FindOne(ctx, byId(c.SectionId()))
```
Comment (john-m-liu) [inline config.go]: Not sure why but when I run this locally this returns no docs found, even though I'm using the same config/db that I've always used

File: cmd/set-var/set-var.go [hunk 2]
```diff
@@ -25,10 +28,17 @@ func main() {
...
-	grip.Infof("set the value of '%s' for document '%s' in collection '%s'", key, id, collection)
+	client, err := mongo.Connect(ctx, options.Client().ApplyURI("mongodb://localhost:27017").SetConnectTimeout(2*time.Second))
+	grip.EmergencyFatal(err)
+	res, err := client.Database(dbName).Collection(collection).UpdateOne(ctx, bson.M{"_id": id}, bson.M{"$set": bson.M{key: value}})
+	grip.EmergencyFatal(err)
+	if res.ModifiedCount != 1 {
+		grip.Warningf("no documents updated: %+v", res)
+		os.Exit(2)
+	}
+	grip.Infof("set the value of '%s' for project '%s'", key, project)
```
[the remark this answers is not in the document]
Comment (tychoish) [reply cmd/set-var/set-var.go]: fixed

File: config.go (hunk 1 above)
Comment (tychoish) [reply config.go]: are you loading from a settings override file? what's the mongodb uri that you're using?

Comment (john-m-liu) [reply config.go]: No file or URI. It should default to localhost:27017 right? That's the port my mongod is running on

**Pull request merged**